app/views/subscriptions/confirm.html.haml

```diff
@@ -3,16 +3,14 @@
 .govuk-grid-row
   .govuk-grid-column-two-thirds
     .govuk-panel.govuk-panel--confirmation
-      %h1.govuk-panel__title= t('subscriptions.confirmation.header')
+      %h1.govuk-panel__title= t('subscriptions.confirmation.header').html_safe
```

Comment:
Why was this change required?

Reply:
Git lens highlighted this change as a part of commit Stuart authored.
 
It looks like it was meant for a change in wording for the subscription header which initially had HTML in it. The change is also a part of the same commit [here](https://github.com/DFE-Digital/teacher-vacancy-service/pull/925/commits/adcc3e0ee4ab00b8eb2883cb936777849650eef1#diff-37c9b835927e3c363588f172de9bb6faR306)
Will remove the `html.safe` since we no longer use HTML in the `subscriptions.confirmation.header` translation file.